src/objects.h

```diff
@@ -232,7 +232,7 @@ enum TNUM {
         // the value for LAST_PACKAGE_TNUM indirectly
         FIRST_PACKAGE_TNUM,
 #ifdef HPCGAP
-        LAST_PACKAGE_TNUM   = FIRST_PACKAGE_TNUM + 41,
+        LAST_PACKAGE_TNUM   = FIRST_PACKAGE_TNUM + 153,
```

Comment:
Can you explain to me (or maybe put in a comment) how this number is derived :)

Reply:
Its the largest value so that the static assert below does not trigger

Reply:
Specifically, this is what the comment right before this line is:
```
        // package TNUMs, for use by kernel extensions
        // note that LAST_COPYING_TNUM must not exceed 253, which restricts
        // the value for LAST_PACKAGE_TNUM indirectly
```
If you have a suggestion how to rephrase that to make it clearer, please let me know.

Reply:
I have expanded the comment substantially, better now?